src/ui/common/src/components/integrations/dialogs/condaDialog.tsx

```diff
@@ -11,10 +11,17 @@ export const CondaDialog: React.FC = ({}) => {
           target="_blank"
           href="https://conda.io/projects/conda/en/latest/user-guide/install/index.html"
         >
-          conda installed
-        </Link>
-        . Once connected, aqueduct server will use conda environment to run your
-        future workflows.
+          conda
+        </Link>{' '}
+        and{' '}
+        <Link
+          target="_blank"
+          href="https://conda.io/projects/conda-build/en/latest/install-conda-build.html"
+        >
+          conda build
+        </Link>{' '}
+        installed. Once connected, aqueduct server will use conda environment to
```

Comment:
Nits: 
* "Aqueduct" instead of "aqueduct server"
* "will use *a* conda environment to run *new* workflows"